modules/distribution-zones/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     annotationProcessor project(":ignite-configuration-annotation-processor")
     annotationProcessor libs.auto.service
 
+    implementation project(':ignite-catalog')
```

Comment:
why do we need this dependency? I would postpone introducing any dependency till the moment whet it actually required